### ISSUE TITLE
Play 2.8 support

### ISSUE THREAD

**Issue by gabfssilva** [iheartradio/play-swagger#298]
Soon Play 2.8 will be released and it'd be nice to have play-swagger supporting it then.

**Comment by vasyl-zhurba** [comment]
Yeah, both sbt plugin and the library

**Comment by kailuowang** [comment]
I am +1 on this. we won't support more than 2 play versions, so we need to stop maintain play 2.6 support on master. Move the current play 2.7 branch to become the new master branch. And maintain 2.8 on a new branch play2.8. If anyone has the time to create a PR against the new created play2.8 branch, I will update master.

**Comment by gabfssilva** [comment]
I'll send the PR right away.

**Comment by lapidus79** [comment]
Any ETA on getting play 2.8.0 version released

**Comment by gabfssilva** [comment]
Fixed after #308
